doc/command_ref.rst

```diff
@@ -411,10 +424,52 @@ Install Command
 ---------------
 
 ``dnf [options] install <spec>...``
-    DNF makes sure that the given packages and their dependencies are installed on the system. Each ``<spec>`` can be
-    either a ``<package-spec>``, which specifies a package directly, or a path to the local rpm package, or an URL to a remote rpm package, or a ``@<group-spec>``, which specifies an (environment) group which contains it. If a given package cannot be (and is not already) installed, the exit code will be non-zero.
+    DNF makes sure that the given packages and their dependencies are installed
+    on the system. Each ``<spec>`` can be either a :ref:`<package-spec>
+    <specifying_packages-label>`, which specifies a package directly (by name,
+    or by name-version, or so on), or a path to the local rpm package, or an URL
+    to a remote rpm package, or a provide which specifies a provide of packages,
+    or a \@\ :ref:`\<group-spec>\ <specifying_groups-label>`, which specifies an
+    (environment) group which contains it. If a given package or provide cannot
+    be (and is not already) installed, the exit code will be non-zero.
+
+    If package with given version is tried to install and other package version
```

Comment:
I like this paragraph - include it with the shorter version of install docs, please. Maybe I would rephrase it a bit:
```
When is given <package-spec> specifying exact version of the package, DNF will install the desired version, no matter which version of the package is already installed. The former version of the package will be removed in case of non-installonly package.